### PR TITLE
Some minor UI fixes

### PR DESCRIPTION
**Pull request by zynskeywolf**
- Simplified the description field in LADSPA Plugin Browser so that it doesn't break all lines at the length of Name
- Changed misleading text in empty automation editor
- Added missing names for controls in some instruments, fixed some existing ones
- Made more space for Spectrum Analyzer led buttons as some long translations didn't fit in

### CODE REVIEW

**Review by Veratil — APPROVED** [review]
Other than a few style fixups that I see, I'd rather just merge this and be good with it. :)

File: plugins/LadspaBrowser/LadspaDescription.cpp [hunk 1]
```diff
@@ -128,37 +128,14 @@ void LadspaDescription::update( const ladspa_key_t & _key )
 	name->setText( QWidget::tr( "Name: " ) + manager->getName( _key ) );
 	layout->addWidget( name );
 
-	auto maker = new QWidget(description);
-	auto makerLayout = new QHBoxLayout(maker);
-	makerLayout->setContentsMargins(0, 0, 0, 0);
-	makerLayout->setSpacing( 0 );
+	auto maker = new QLabel(description);
+	maker->setText(QWidget::tr("Maker: ") + manager->getMaker(_key));
```
Comment (sakertooth) [inline plugins/LadspaBrowser/LadspaDescription.cpp:132]: We should be using `LadspaDescription::tr` here as Qt translations allow us to specify a context for the translation, and putting it in the context of `LadspaDescription` is more specific and relevant.

```suggestion
	maker->setText(tr("Maker: 2") + manager->getMaker(_key));
```

File: plugins/LadspaBrowser/LadspaDescription.cpp [hunk 2]
```diff
@@ -128,37 +128,14 @@ void LadspaDescription::update( const ladspa_key_t & _key )
...
-	maker_content->setWordWrap( true );
-	makerLayout->addWidget( maker_label );
-	makerLayout->addWidget( maker_content, 1 );
-
-	auto copyright = new QWidget(description);
-	auto copyrightLayout = new QHBoxLayout(copyright);
-	copyrightLayout->setContentsMargins(0, 0, 0, 0);
-	copyrightLayout->setSpacing( 0 );
+	auto copyright = new QLabel(description);
+	copyright->setText(QWidget::tr("Copyright: ") + manager->getCopyright(_key));
```
Comment (sakertooth) [inline plugins/LadspaBrowser/LadspaDescription.cpp:136]:
```suggestion
	copyright->setText(tr("Copyright: ") + manager->getCopyright(_key));
```

File: plugins/LadspaBrowser/LadspaDescription.cpp (hunk 1 above)
Comment (zynskeywolf) [reply plugins/LadspaBrowser/LadspaDescription.cpp]: Done.

**Review by sakertooth — APPROVED** [review]

**Pull request merged**